WORKSPACE

```diff
@@ -141,6 +141,21 @@ http_archive(
 
 load("@rules_jvm_external//:defs.bzl", "maven_install")
 
+ATS_TAG = "1edfdab3134a7f01b37afabd3eebfd2c5bb05151"
```

Comment:
Why not use a specific version, instead?

Reply:
I didn't get it can you provide some details regarding this.

As per the ATS_TAG I've taken the [BasicSample app](https://github.com/android/testing-samples/blob/main/WORKSPACE#L10) as an example